client/app/recordings/new/route.js

```diff
@@ -7,11 +7,62 @@ export default Route.extend({
     this.container = options.container;
   },
 
-  render() {
-    this.view = new View({
-      model: new Model(),
-      editing: true
-    });
-    this.container.show(this.view);
-  }
+  render(params) {
```

Comment:
So the pattern I've been using thus far in this codebase is that this method should 1) kick off some set of xhr's (through `#fetch` on models or collections) 2) instantiate a view with the models that have just been fetched.